ui/static/ui/js/lr_exports.jsx

```diff
@@ -0,0 +1,77 @@
+define("lr_exports",
+  ['reactaddons', 'jquery', 'lodash', 'utils'], function (React, $, _, Utils) {
+  'use strict';
+
+  var ExportsComponent = React.createClass({
+    componentDidMount: function() {
```

Comment:
I wonder if there is a way to document ReactJS component classes that doesn't bloat the code that goes over the wire?  It doesn't look like JSX strips comments when it compiles to javascript.